packages/react-components/react-drawer/src/components/DrawerHeaderTitle/DrawerHeaderTitle.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { useDrawerHeaderTitle_unstable } from './useDrawerHeaderTitle';
+import { renderDrawerHeaderTitle_unstable } from './renderDrawerHeaderTitle';
+import { useDrawerHeaderTitleStyles_unstable } from './useDrawerHeaderTitleStyles.styles';
+import type { DrawerHeaderTitleProps } from './DrawerHeaderTitle.types';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useCustomStyleHook_unstable } from '@fluentui/react-shared-contexts';
+import { useDialogTitleStyles_unstable } from '@fluentui/react-dialog';
+
+/**
+ * DrawerHeader provides a structured header for the drawer component.
+ */
+export const DrawerHeaderTitle: ForwardRefComponent<DrawerHeaderTitleProps> = React.forwardRef((props, ref) => {
+  const state = useDrawerHeaderTitle_unstable(props, ref);
+
+  const { heading: root = {}, action, components } = state;
+
+  useDialogTitleStyles_unstable({
+    components: {
+      root: components.heading,
+      action: components.action,
+    },
+    root,
+    action,
+  });
```

Comment:
You should call this hook from within `useDrawerHeaderTitleStyles` so that it still follows our composition mode - one style hook per component. In this case you have 2 and one doesn't belong in this package. Only what is internally within the hooks are really considered internals

Reply:
Makes sense. Fixed now.